proposals/PROPOSALS.md

```diff
@@ -1,10 +1,29 @@
-An HFTP (*Haskell Foundation Technical Proposal*) is a process for submitting a proposal for a technical initiative to that will be executed under the guidance of the Haskell Foundation (HF). This document describes how the community may propose, discuss, and implement technical initiatives to be carried out through the Haskell Foundation that will affect the Haskell ecosystem. Each HFTP goes through an iterative process in which it is discussed by the HF Technical Track (HFTT), the broader community, and any and all interested stakeholders. If a consensus is reached, the initiative is accepted and may then be executed by the Foundation (resources permitting). Only upon reaching a consensus are initiatives accepted and executed. Upon acceptance, an HFTP and its associated discussion are merged into this repository, and both serve as a historical document that details a specification and rationale for work detailed within the process.
```

Comment:
Please can we number the sections and sub-sections.  

I know that (sadly) Markdown does not support this, and I know it's painful to do manually.  But it's *more* painful not to be able to say "In section 2.2 ....".

Reply:
I wrote the 20 lines of elisp that were required, and it works, but on further reflection, I'm a bit hesitant to impose bureaucratic maintenance requirement on further editors of a document.  It might be best to just chalk this up to the limitations of Markdown, rather than constantly fighting the format.